docs/docs/references/users/README.md

```diff
@@ -261,3 +261,49 @@ Every time the user signs in with a social connector,
 their _identities_ will be automatically imported or updated from the identity provider.
 
 :::
+
+## Session and cookie
+
+It's inconvenient to sign in every time the user re-visits the application.
+The application using Logto can extend the session by putting its information in
+[cookies](https://en.wikipedia.org/wiki/HTTP_cookie) for reuse to avoid this inconvenience.
+The extended session will end when it expires, the user signs out on purpose, or the password has been changed.
+
+### Session lifetime
+
+The Logto session's _lifetime_ is 14 days by default, following the
+[oidc-provider TTL](https://github.com/panva/node-oidc-provider/blob/main/docs/README.md#ttl)
+([Time To Live](https://en.wikipedia.org/wiki/Time_to_live)).
+After the session expires (i.e., its _lifetime_ finishes), the user must sign in again.
+
+### Session layers
+
+Using Logto as an identity provider, three layers of sessions may be created:
+
+- **Application Layer**: When a user signs in using Logto and goes to your application, you need to keep track of the authentication info, such as the "authenticated" state.
+  Keeping track of them usually means saving a set of tokens in the storage provided by your application.
+  The storage can be cookies in a regular web application.
+- **Logto Layer**: Logto keeps track of the session through the browser's cookies.
+  After the session expires, the user may be redirected to Logto sign-in page and need to enter the authentication info again.
+- **Social Sign-in Layer**: When the user is redirected to Logto sign-in page and tries to sign in with a social connector, an additional redirection to the social identity provider is needed.
```

Comment:
we don't care about social sign-in here. you were talking about "Logto layer session" in the previous section but talks a broader "session" here. very confusing